src/main/java/uk/gov/hmcts/darts/retention/enums/RetentionConfidenceScoreEnumConverter.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.darts.retention.enums;
+
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+import java.util.stream.Stream;
+
+@Converter(autoApply = true)
+public class RetentionConfidenceScoreEnumConverter implements AttributeConverter<RetentionConfidenceScoreEnum, Integer> {
+
+    @Override
+    public Integer convertToDatabaseColumn(RetentionConfidenceScoreEnum retentionConfidenceScoreEnum) {
+        if (retentionConfidenceScoreEnum == null) {
+            return null;
```

Comment:
Instead of returning null, would it make sense to use 0 as NOT_SET for instance?

Reply:
I don't think setting 0 for absence of value is correct. absence of value should be represented with a `null`